cogs/misc.py

```diff
@@ -39,80 +34,20 @@
 # in case multiple streams are directed to the same file
 LOG_FILES = {
     STDERR_LOG_FILE: "All standard errors are redirected here.",
-    "log.txt": "All Logging.Log() lines are saved here.",
+    "log.txt": "All Logging.log() lines are saved here.",
```

Comment:
```suggestion
    "log.txt": "All logger.log() lines are saved here.",
```

Reply:
thx, fixed in 6f31f0b0cbb737193324995a9c7691ca87f9b79a